cogs/channels.py

```diff
@@ -1,42 +1,60 @@
 import discord
 from discord.ext import commands
-from discord.ext.commands import group
 from cogs.base_cog import BaseCog
 
 
 class Channels(BaseCog):
-    def __init__(self, bot):
-        super().__init__(bot)
-
-    @group()
-    async def tch(self, ctx):
+    @commands.group(invoke_without_command=True)
+    async def channel(self, ctx):
         """Text Channel edition commands."""
-        if ctx.invoked_subcommand is None:
-            await self.bot_reply(ctx, 'Invalid Text Channel Edition command passed, {}'.format(ctx.author.mention))
+        await ctx.invoke(self.bot.get_command("help"), ctx.invoked_with)
+
+    @channel.group(description='Create new channels with Supervisor permission.', invoke_without_command=True)
+    @commands.has_any_role('Nixie', 'Supervisors')
+    @commands.guild_only()
+    async def create(self, ctx):
+        await ctx.invoke(self.bot.get_command("help"), ctx.command.parent.name, ctx.invoked_with)
 
-    @tch.command(description='Create a new text channel with Supervisor permission.')
+    @create.command(description='Create a new text channel with Supervisor permission.')
```

Comment:
why @create vs @tch? API Changes?

Reply:
i thought it would be clearer for usage, `@create text ...` vs `@tch create_text ...` or whatever it was

Reply:
no worries, just curious if there was a behavioral change.